documentation/SDKs/javascript/README.md

```diff
@@ -0,0 +1,358 @@
+#JavaScript SDK for Nether
+
+## Work in Progress
+
+## Downloading the JavaScript SDK for Nether
+
+### Installing the JavaScript SDK for Nether
+
+## Usage
+
+### Analytics
+
+#### Active sessions - daily
+You can use the ***Nether.Analytics.GetActiveSessionsDaily*** method to get the total active sessions per day. Returns an array of ActiveSession
```

Comment:
There are a *lot* of methods for analytics and we plan on adding more analytics types. Should we put in a grouping here?

e.g.

```javascript
 Nether.Analytics.ActiveSessions.GetDaily();
```

or 

```javascript
 Nether.Analytics.GetActiveSessions( {"granularity": "dailly"});
```

@krist00fer  - thoughts?